tests/spec/core/pluralize-spec.js

```diff
@@ -189,23 +189,6 @@ describe("Core - Pluralize", () => {
     ).toBeTruthy();
   });
 
-  it("doesn't add pluralization when no <a> references plural term", async () => {
```

Comment:
any reason to remove this test?

Reply:
Lets just add a test `"doesn't add singularization when no <a> references singular term"` and keep this here

Reply:
Doesn't make sense. Essentially now either pluralization or singularization will be applied ( both being called Pluralization). We will always have one form of pluralization, which is ensured in the first test itself.

Reply:
My Bad, I seem to have made a very bad error, I accidentally pushed it out of the describe block. Correcting.

Reply:
We still need to ensure that both paths in code are tested.
So, lets try these:
``` html
<dfn data-lt="baz">bar</dfn> can be referenced as <a>baz</a> or <a>bar</a> 
so dataset.plurals should be empty as we're not referencing "bazs" or "bars"
```

``` html
<dfn data-lt="tables">chairs</dfn> can be referenced as <a>chairs</a> or <a>tables</a> 
so dataset.plurals should be empty as we're not referencing "table" or "chair".
```

But in above, if we respectively refer to plural/singular terms also, we should have them in `dataset.plurals`. Makes sense?

Reply:
Yes, I did get your point. Both path are technically the same, so the same test works for both. but I will add the test as a good security measure. Really appreciate going out of your way to make things clear :)